src/devices/Dhtxx/Devices/IDhtDevice.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+
+namespace Dhtxx.Devices
+{
+    public interface IDhtDevice
```

Comment:
I suppose that this is the only two things that all dht devices can do? if there are any Dhtxx that have more functionality than these two things, I would suggest using an abstract base class instead of interface, since abstract base class can add members in the future, but interface can't.

Reply:
At present, the DHT series sensors only have the function of temperature and humidity, and the DHT series has not been updated in these years. Sensors with other functions are not named DHT by ASAIR. So I think all of them can.

Reply:
Well i can imagine that we have to add some generic methods for all devices in the future. So an abstract class wouldn’t be an bad idea.

Reply:
I'm fine with this as is today, so I'll go ahead and merge in order for people currently hitting an issue with this binding to have a new version that works. We can decide later if we want to have an abstract base class instead of an interface.